d3-series/materials/Week 6/Assignments/us-station.html

```diff
@@ -0,0 +1,174 @@
+<html>
+
+    <head>
+        <title>Map</title>
+    </head>
+
+    <body>
+        <div id="map-holder"></div>
+
+        <script src="https://d3js.org/d3.v4.min.js"></script>
+        <script>
+            function buildMap(containerId) {
+                // size globals
+                var width = 960;
+                var height = 600;
+
+                var margin = {
+                    top: 80,
+                    right: 50,
+                    bottom: 50,
+                    left: 50
+                };
+
+               
+                var innerWidth = width - margin.left - margin.right;
+                var innerHeight = height - margin.top - margin.bottom;
+
+                var svg = d3
+                    .select(containerId)
+                    .append('svg')
+                    .attr('height', height)
+                    .attr('width', width);
+
+           
+                var g = svg
+                    .append('g')
+                    .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
+
+    
+                d3.json('us-states.json', function(error, geojson) {
+                    handleError(error, 'failed to read geoJSON data');
+
+                    d3.csv('NSRDB_StationsMeta.csv', function(error, stations) {
+                        handleError(error, 'failed to read station data');
+                        stations=prepDataStations(stations);
+                        console.log(stations);
+                        draw(geojson, stations);
+                        
+                        
+                    });
+                });
+
+                function handleError(error, msg) {
+                    if (error) {
+                        console.error(msg);
+                    }
+                }
+
+                function prepDataStations(data) {
+                    return data
+                    .map(function(d) {
+                        return {
+                            name: d.USAF,
+                            class: +d.CLASS,
+                            loc: [+d.longitude, +d.latitude],
+                            ele: +d.NSRDB_ELEV

```

Comment:
It looks like this should be +d['NSRDB_ELEV (m)'] as the data prop name. Once I do this the map renders!